.github/workflows/validate.yaml

```
@@ -2,16 +2,40 @@ name: Validate
 
 on:
   push:
-  pull_request:
-  schedule:
-    - cron: "0 0 * * *"
 
 jobs:
   validate:
+    name: Validate HACS App
```

I thought about adding a condition to run it only on master branch, because the action fails on forked repo: https://github.com/ericmatte/ad-spotify-mood-lights-sync/runs/1958334507?check_suite_focus=true
```suggestion
    if: github.ref == 'refs/heads/master'
    name: Validate HACS App
```

---

So I've found a better solution 🎉 https://github.com/ericmatte/ad-spotify-mood-lights-sync/runs/1958419591?check_suite_focus=true

Using this:
```yaml
      - name: Ignore issues and topics for non-main branch
        if: github.ref != 'refs/heads/master'
        run: echo 'IGNORE=issues topics' >> "$GITHUB_ENV"

      - name: HACS validation
        uses: "hacs/action@main"
        with:
          category: "appdaemon"
          comment: "false"
          ignore: "${{ env.IGNORE }}"
```